docs/backport_release.md

```diff
@@ -0,0 +1,131 @@
+3. Add yourself to the list of contributors to this release and **put the contributors in alphabetical order**
+4. The release PR does not need to be mentioned in the list of changes
+5. Add a commented out "Future Release" section with all of the Release Notes sections above the current section
+
+   ```
+   .. Future Release
+     ==============
+       * Enhancements
+       * Fixes
+       * Changes
+       * Documentation Changes
+       * Testing Changes
+
+   .. Thanks to the following people for contributing to this release:
+   ```
+
+#### Create Release PR
+
+A [release pr](https://github.com/alteryx/featuretools/pull/1915) should have the version number as the title and the release notes for that release as the PR body text. The contributors list is not necessary. The special sphinx docs syntax (:pr:\`547\`) needs to be changed to github link syntax (#547).
+
+Checklist before merging:
+
+- All tests are currently green on checkin and on `0.11.x`.
+- The ReadtheDocs build for the release PR branch has passed, and the resulting docs contain the expected release notes.
+- PR has been reviewed and approved.
+- Confirm with the team that `0.11.x` will be frozen until step 2 (Github Release) is complete.
+
+## 2. Create Github Release
+
+After the release pull request has been merged into the `0.11.x` branch, it is time draft the github release. [Example release](https://github.com/alteryx/featuretools/releases/tag/v1.6.0)
+
+- **The target should be the `0.11.x` backport branch**
+- The tag should be the version number with a v prefix (e.g. v0.11.2)
+- Release title is the same as the tag
+- Release description should be the full Release Notes updates for the release, including the line thanking contributors. Contributors should also have their links changed from the docs syntax (:user:\`gsheni\`) to github syntax (@gsheni)
+- This is not a pre-release
+- Publishing the release will automatically upload the package to PyPI
+
+Note that this backported release will show up on the repository's front page as the latest release even if there is technically a later `0.12.0` release.
+
+## Release on conda-forge
+
+If a later release exists, conda-forge will not automatically create a new PR in [conda-forge/featuretools-feedstock](https://github.com/conda-forge/featuretools-feedstock/pulls). Instead a PR will need to be manually created. You can do either of the following:
+
+- Branch off of the 0.11.1 meta.yaml update commit for the 0.11.2 meta.yaml changes. This is "cleaner" and sometimes easier, but if migration files (like py310) have been added between 0.11.1 and 0.12.0 you will have to add them in and re-render yourself.
+- Tack the 0.11.2 changes on after the 0.12.0 update commit in the feedstock repo. This means that if any of the boilerplate has changed, you do not have to manually re-add it yourself. An example of this can be seen from a Woodwork backport release [here](https://github.com/conda-forge/woodwork-feedstock/pull/32).
```

Comment:
Do you have a Featuretools example to link to?

Reply:
the recent featuretools backport wasn't actually between two release numbers, so conda forge didn't have any problems. I don't know of any other times we've backport released featuretools

Reply:
np.